content/docs/gto/user-guide/dvc.md

```diff
@@ -0,0 +1,144 @@
+# GTO with DVC
+
+In many ML projects, data isn't stored in a Git repository and needs to be
+downloaded from external sources. [DVC](https://dvc.org) is a common way to
+store binaries for the artifacts registered with GTO.
+
+<details>
+
+### Learn about different approaches to this
+
+1. You can commit artifacts to Git repo. If they aren't small enough, this is
+   not recommended. To bypass this limitation, you can use
+   [Git-lfs](https://git-lfs.github.com).
+2. You can version binaries with [DVC](https://dvc.org/) and commit pointers to
+   them to the repo. This is the recommended approach for large files.
+3. You can version binaries manually somewhere, specifying URL to them as `path`
+   in `artifacts.yaml`. This can be done, if your artifacts are already
+   versioned by some external system.
+
+</details>
+
+If you are new to DVC, you should start with
+[DVC Get Started](https://dvc.org/doc/start) first, and then come back to this
+Guide.
+
+<!-- ```
+
+dvc init --no-scm dvc remote add az azure://container-name export
+AZURE_STORAGE_CONNECTION_STRING='YOUR_CONNECTION_STRING'
+
+dvc import-url --no-download azure://container-name/data.parquet
+
+``` -->
+
+## Tracking an artifact with DVC
+
+First, we need to start tracking artifact with DVC. If you produce this artifact
```

Comment:
I don't think we need this section at all: it's DVC documentation (redundant since we already linked to the Get Started)...

Reply:
Besides, this shows `dvc import-url`, which could be an important scenario for some folks starting to use MR while having some models outside.

Reply:
People without context will get the bare minimum from this part. People with context will just see some story here.

Reply:
WDYT?

Reply:
I think that linking to https://dvc.org/doc/start/data-management/data-versioning or even straight to https://dvc.org/doc/command-reference/add would be acceptable.

Not a super strong opinion since this section is small, but I think that in principle we should not cross-document products.

_Logic: what if the DVC feature changes? dvc.org/doc will probably get updated, but not here._

> Besides, this shows dvc import-url

Not seeing why that's particularly important. BTW, the blocks are missing `git add .gitignore`